lib/iso8583/lib/packer/fixed-b-hex.js

```diff
@@ -26,6 +26,8 @@ exports.unpack = function(msg, packager) {
     data += chunk;
     bitmap += chunkBitmap;
   }
+  
+  data = parseInt(data.replace(/^0+/, ''),16)
```

Comment:
Can't get the idea why heading spaces need to be removed?

Reply:
It is because it is casted to an integer.

Example:
<Buffer 07 d0> -> 0x7d0 = Int 2000

Reply:
Yes, but what is the difference?

console.log(parseInt('0x07d0',16));
console.log(parseInt('07d0',16));
console.log(parseInt('7d0',16));

node t.js                                                                                                                                                      2000                                                                                                                                                                         2000                                                                                                                                                                         2000

Reply:
You are right, its modified now.